src/main/scala/cromwell/engine/db/slick/SlickDataAccess.scala

```diff
@@ -227,10 +227,13 @@ class SlickDataAccess(databaseConfig: Config, val dataAccess: DataAccessComponen
     // Insert an execution row
       executionInsert <- dataAccess.executionsAutoInc +=
         new Execution(
-          workflowExecution.workflowExecutionId.get,
-          key.scope.fullyQualifiedName,
-          key.index.fromIndex,
-          ExecutionStatus.NotStarted.toString)
+          workflowExecutionId = workflowExecution.workflowExecutionId.get,
+          callFqn = key.scope.fullyQualifiedName,
+          index = key.index.fromIndex,
+          status = ExecutionStatus.NotStarted.toString,
+          rc = None,
+          startDt = new Date().toTimestamp,
```

Comment:
Yes, since Running appears to be set at the appropriate point in the lifecycle of a call, it looks like that should work.  There is some craftiness required for the update to target the right column under the right circumstances, but that can be done.